net-libs/signon-oauth2/signon-oauth2-9999.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
This one is a mess because `net-libs/signond` installs files referencing Qt 6, like:
```
$ cat /usr/lib64/pkgconfig/signon-plugins.pc
prefix=/usr
exec_prefix=${prefix}
libdir=${prefix}/lib64
includedir=${prefix}/include
plugin_includedir=${includedir}/signon-plugins/SignOn
plugindir=/usr/lib64/signon
Name: signon-plugins
Description: SignonPlugins
Version: 8.61
Requires: libsignon-qt6
Libs.private: -L${libdir} -lQt6Core
Cflags:  -I${includedir}/signon-plugins -I${includedir}/signon-plugins/SignOn
Libs: -lsignon-plugins

$ cat /usr/lib64/pkgconfig/SignOnExtension.pc
prefix=/usr
exec_prefix=${prefix}
libdir=${prefix}/lib64
includedir=${prefix}/include
plugindir=/usr/lib64/signon/extensions

Name: SignOnExtension
Description: SignOn extension development
Version: 8.61
Requires: Qt6Core
Libs: -L${libdir} -lsignon-extension
Cflags: -I${includedir}/signon-extension
```

Reply:
The problem is, even if say, we make the .pc files prefixed/suffixed with the Qt version, that makes us incompatible with other distros and is very ungentoo.

So we're down to:
1) Do upstream care about co-installability here?
2) Can we get away with only allowing qt5 or qt6?

Reply:
thanks, done at https://gitlab.com/accounts-sso/signon-plugin-oauth2/-/merge_requests/28#note_1687358193

Reply:
He's replied so I'll try look at that tomorrow.

Reply:
https://github.com/thesamesam/kde/tree/gear-24.02-prep may be good enough here now. The only real question in my head is whether we should try to split signond (hopefully not) but I _think_ the dependencies enforcing the matching Qt version in plugins are enough.